.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "printWidth": 80,
+  "printWidth": 100,
```

Comment:
I increased the wrap width to 100 chars, and some of the files have reformatting in them.